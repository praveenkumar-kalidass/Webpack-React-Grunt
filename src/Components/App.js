import React from "react";

class AppContainer extends React.Component {
  render() {
    return (
      <div>
        My App
      </div>
    );
  }
}

export default AppContainer;
