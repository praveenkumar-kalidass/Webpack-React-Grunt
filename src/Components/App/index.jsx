import React from "react";
import {Button} from "react-bootstrap";
import "./style.scss";

class App extends React.Component {
  render() {
    return (
      <div className="app">
        My App
        {this.props.children}
      </div>
    );
  }
}

export default App;
