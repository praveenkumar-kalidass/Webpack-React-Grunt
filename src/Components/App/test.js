import React from "react";
import App from "./index";
import renderer from "react-test-renderer";

test("App Container", () => {
  const component = renderer.create(
    <App />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
