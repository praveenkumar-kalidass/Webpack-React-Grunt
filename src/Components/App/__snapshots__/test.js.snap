// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Container 1`] = `
<div
  className="app"
>
  My App
</div>
`;
